.gitignore

```diff
@@ -76,7 +76,9 @@ doc/docsrc/*.xdv
 /examples/sample.source.*/lib*.a
 /examples/sample.source.*/.tmp.*
 /examples/sample.source.*/Makefile.am.libasncodec
+/examples/sample.source.*/libasncodec.am
```

Comment:
What's your rationale for changing the makefile names?

Reply:
I changed the Makefile.am.example to example.mk to reflect that it is a *plain* makefile snippet rather than a file that shows the characteristics of a makefile snippet that needs to be transformed via automake. Also, *.mk keeps the syntax highlighting in most editors.
This rationale doesn't hold as much water in regards to the Makefile.am.libasncodec though. I will undo these changes.

Reply:
Perhaps the least surprising name would be `converter-example.mk`. This way, there's no question about what does what.